web-api/openapi/schemas/user.yml

```diff
@@ -2,3 +2,23 @@ UserId:
   description: Id of a user.
   $ref: './id.yml#/Id'
   example: "user_id"
+
+InputUser:
+  description: |
+    Information about a user provided as input for an search
+
+    This can be anything from no user information, to a user id, to
```

Comment:
This should represent `no user information`, which is already represented by having the field `user` optional in `SemanticSearchrequest`.